template/scripts/generateDocs.js

```diff
@@ -1,14 +1,84 @@
-const path = require('path')
+const path = require('path');
 const chalk = require('chalk');
-const markdownMagic = require('markdown-magic')
+const markdownMagic = require('markdown-magic');
+const fs = require('fs');
+const https = require('https');
 
-const config = {
-  matchWord: 'AURO-GENERATED-CONTENT'
+const readmeTemplateUrl = 'https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator/master/componentDocs/README.md'
+
+// Copy README.md template from static source
+const dir = './docTemplates';
+
+if (!fs.existsSync(dir)){
+  fs.mkdirSync(dir);
+}
+
+const readmeFilePath = './docTemplates/README.md';
+
+if (!fs.existsSync(readmeFilePath)) {
+  fs.writeFile(readmeFilePath, '', function(err) {
+    if(err) {
+      console.log('Unable to create README.md file', err);
+    }
+  });
+}
+
+const readmeTemplateFile = fs.createWriteStream(readmeFilePath);
+const request = https.get(readmeTemplateUrl, function(response) {
+  response.pipe(readmeTemplateFile);
+});
+
+// Compile `./docTemplates/README.md` -> `./README.md`
```

Comment:
Is this commented out code or a comment of the code? 

If a comment of the code, please use the `/** */ syntax.